sdk/formrecognizer/ai-form-recognizer/recordings/node/aad_formrecognizerclient_nodejs_only/recording_recognizes_content_from_a_url.js

```diff
@@ -1,6 +1,6 @@
 let nock = require('nock');
```

Comment:
The changes in the recordings doesn't seem to have requests deleted.

Reply:
Done 👀